app/views/_user_util_links.html.erb

```diff
@@ -2,12 +2,12 @@
 <div class="row-fluid">
 <ul id="user_utility_links" class="nav navbar-nav navbar-right">
   <% if user_signed_in? %>
-  <% if current_user.admin? %>
   <li>
-  
+    <%= render_notifications(user: current_user) %>
   </li>
+  <% if current_user.admin? %>
   <li>
-      <%= render 'hyrax/users/notify_number' %>
+
   </li>
   <% end %>
```

Comment:
I'm not sure I understand what lines 8-12 are supposed to do, they just render as an empty 0px X 0px list item when I run the application.

Reply:
@mark-dce That's the most technical description for an empty div I've ever read. 💯 

I removed it in a rebase, and the `admin` block. Non-admin users will need to a way to see notifications in the dashboard – right?